app/models/illinois_rate.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 # Subsidy rules that apply for Illinois
-class IllinoisSubsidyRule < UuidApplicationRecord
-  has_one :subsidy_rule, as: :subsidy_ruleable, dependent: :restrict_with_error
+class IllinoisRate < UuidApplicationRecord
+  has_one :rate, as: :state_rule, dependent: :restrict_with_error
```

Comment:
```suggestion
  has_one :rate, as: :rateable, dependent: :restrict_with_error
```

Couple reasons for this suggestion:
- it's "standard" syntax for polymorphics
- the state rule *is* the IllinoisRate; the polymorphic rate table record (which table will be deprecated eventually) is the generalized rate info that has nothing to do with the state.